src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -465,6 +465,117 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
 
+
+  #
+  # Anonymous Elector Document Applications
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-applications':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application(s).
+        schema:
+          type: string
+        example: 'city-council'
+        in: path
+        required: true
+    options:
+      summary: CORS support
+      description: |
+        Enable CORS by returning correct headers
+      tags:
+        - Anonymous Elector Document Applications
+      responses:
+        200:
+          description: Default response for CORS method
+          headers:
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+          content: { }
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: |
+                {}
+    get:
+      summary: Search for Anonymous Elector Document Applications
+      description: 'Search for Anonymous Elector Document Applications based on querying parameters'
+      tags:
+        - Anonymous Elector Document Applications
+      responses:
+        '200':
+          $ref: '#/components/responses/SearchApplications'
+      operationId: search-anonymous-elector-document-applications
+      parameters:
+        - name: pageSize
+          description: The number of results to return in each page
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 200
+          example: 100
+          in: query
+          required: true
+        - name: page
+          description: The page number being requested (with the first page being 1)
+          schema:
+            type: integer
+            minimum: 1
+          example: 1
+          in: query
+          required: true
+        - name: searchBy
+          description: |
+            The data field to apply the `searchValue` to. An empty value will return all AED applications.
+            This value must be provided when `searchValue` is present
+          schema:
+            $ref: '#/components/schemas/SearchBy'
```

Comment:
Nice idea, but this is really for future stories (EIP1-5591 onwards) 👍

Reply:
These are optional fields and ui can just choose to ignore them until we have implemented these in api